securedrop/source_templates/base.html

```diff
@@ -18,33 +18,33 @@
 
     <div class="content">
       <div class="container">
+        <header>
         {% block header %}
-        <div id="header">
           <a href="{% if 'logged_in' in session %}{{ url_for('main.lookup') }}{% else %}{{ url_for('main.index') }}{% endif %}" class="no-bottom-border">
-            <img src="{{ g.logo }}" class="logo small" alt="{{ g.organization_name }} | {{ gettext('SecureDrop Home') }}" width="250">
+            <img src="{{ g.logo }}" class="logo small" alt="{{ g.organization_name }} logo" width="250">
           </a>
           {% include 'locales.html' %}
-        </div>
         {% endblock %}
+        </header>
 
-        <div class="panel selected">
+        <main>
           {% if g.show_offline_message %}
+          <section>
           <h1>{{ gettext("We're sorry, our SecureDrop is currently offline.") }}</h1>
           <p>{{ gettext("Please try again later. Check our website for more information.") }}</p>
+          </section>
           {% else %}
 
           {% if 'logged_in' in session %}
-            <a href="{{ url_for('main.logout') }}" class="btn pull-right" id="logout">{{ gettext('LOG OUT') }}</a>
-          {% endif %}
-
-          {% if not new_user%}
-            <hr class="no-line">
+            <nav>
+              <a href="{{ url_for('main.logout') }}" class="btn pull-right" id="logout">{{ gettext('LOG OUT') }}</a>
```

Comment:
I think the use of upper case in the UI is indeed a bigger design question; see also the similar #5821. If we do want to stick with all-caps UI elements, is there a way to instruct screenreaders to not read them out as abbreviations?

Reply:
Completely agree that it's a bigger design question since uppercases are being used in other places as well, as mentioned in #5821 . If it's shown in UI as all-caps, then it's completely on the assistive technology how they will interpret that.

Technically, you can change the actual text in HTML to title case, and then use CSS `text-transform: uppercase` to convert it to uppercase, which works in some assitive techs, but for most it doesn't. The assistive techs try to be as close to what is showing in UI as possible. Now usually if it's a small word, and especially if there is an abbreviation with same characters, it will mostly read out as abbreviation for sure. But for bigger words, most probably they will read it fine. Some screen reader also have a setting where there is a change of tone to denote that they are reading out an all-caps part of the sentence. So it really depends on how the screen reader is implemented, but they try to be as close to the UI shown in text as possible.

But if you really really want absolutely the behaviour where you want all-caps in UI and title case in screen reader, one workaround is to add `aria-label="Log Out"`, though it's usually recommended to avoid doing that.

Some similar discussions that might help:
https://www.drupal.org/project/drupal/issues/2958239
https://groups.google.com/a/chromium.org/g/chromium-accessibility/c/enk1PBjEfRc/m/-npdRy3qyYcJ

Reply:
Thanks for the context and research, @SaptakS!  The discussions you cite are among the sources I consulted in https://github.com/freedomofpress/securedrop/issues/6003#issuecomment-880898715, where I arrived at the same conclusion for the purpose of this PR, pending #5821.  It sounds like you would recommend erring on the side of including these `aria-label`s on all-caps elements; I'll include those changes (to this instance and elsewhere) in my revision based on your review.

Reply:
79b592499 adds these `aria-label`s.